test/Attenuation_test.cpp

```diff
@@ -0,0 +1,180 @@
+#include <iostream>
+#include <normal_depth_map/ImageViewerCaptureTool.hpp>
+#include <normal_depth_map/NormalDepthMap.hpp>
+#include <normal_depth_map/Tools.hpp>
+
+#include <osg/Geode>
+#include <osg/Group>
+#include <osg/ShapeDrawable>
+#include <osgDB/ReadFile>
+
+#include <opencv2/contrib/contrib.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+
+#define BOOST_TEST_MODULE "Attenuation_test"
+#include <boost/test/unit_test.hpp>
+
+using namespace normal_depth_map;
+
+BOOST_AUTO_TEST_SUITE(test_NormalDepthMap)
+
+BOOST_AUTO_TEST_CASE(attenuationCalculation_testCase){
+    double frequency = 700.0;   // kHz
+    double temperature = 20.0;  // celsius degrees
+    double depth = 1;           // meters
+    double salinity = 35;       // ppt
+    double acidity = 8.1;       // pH
+
+    double attenuationCoeff = underwaterSignalAttenuation(frequency, temperature, depth, salinity, acidity);
+    BOOST_CHECK_CLOSE(attenuationCoeff, 0.0247, 3);
+}
+
+// simple sonar plot
+void plotSonarTest(cv::Mat3f image, double maxRange, double maxAngleX) {
```

Comment:
It is better make a new CPP file with all methods required to support tests.

Reply:
Ok.